internal-tooling/performanceTest.ts

```diff
@@ -37,8 +41,14 @@ export const enum TRANSFER_MANAGER_TEST_TYPES {
   APPLICATION_DOWNLOAD_MULTIPLE_OBJECTS = 'application-download',
 }
 
+const enum OUTPUT_FORMATS {
+  CSV = 'csv',
+  SSB = 'ssb', //Storage shared benchmarking format
```

Comment:
super nit, but IIUC the format is related to Cloud Monitoring, and not necessarily the SSB?

Reply:
I had no idea this was a standardized format. I will adjust naming accordingly.